core/templates/dev/head/pages/question_editor/QuestionEditorDirective.js

```diff
@@ -25,7 +25,7 @@ oppia.directive('questionEditor', [
         getQuestionId: '&questionId',
         getMisconceptions: '&misconceptions',
         canEditQuestion: '&',
-        questionStateData: '='
+        question: '='
```

Comment:
If you are doing this, might as well remove questionId above as well, and use the question object itself?
Also, did you change all instances of this directive to use this? I think this is used in creator_dashboard as well.